proto/identity/associations/signature.proto

```diff
@@ -53,7 +55,7 @@ message Signature {
   //    must be inferred from the context in which the signature is provided.
   oneof signature {
     RecoverableEcdsaSignature erc_191 = 1;
-    Erc1271Signature erc_1271 = 2;
+    SmartContractWalletSignature erc_6492 = 2;
```

Comment:
The name here is `erc_6492`, but it's not necessarily a 6492 signature right?

Reply:
That's right, basically 6492 handles 1271 & EOA signatures as well.